skypy/pipeline/_config.py

```diff
@@ -81,6 +81,14 @@ def construct_function(self, name, node):
 
         return (function, args, kwargs)
 
+    def construct_lambda(self, node):
+
+        lambda_val = node.value
+
+        assign_lambda_to_temp = 'temp = lambda {}'.format(lambda_val)
+        exec(assign_lambda_to_temp, globals())
+        return temp
```

Comment:
```suggestion
        lambda_val = 'lambda ' + self.construct_scalar(node)
        return eval(lambda_val, {})
```

Simpler construction that also warns if a sequence or mapping node is given.